addons/editor/functions/fnc_initDisplayCurator.sqf

```diff
@@ -50,6 +50,18 @@ if (GVAR(disableLiveSearch)) then {
     _ctrl ctrlAddEventHandler ["ButtonClick", {call FUNC(fixSideButtons)}];
 } forEach IDCS_MODE_BUTTONS;
 
+private _ctrlTreeRecent = _display displayCtrl IDC_RSCDISPLAYCURATOR_CREATE_RECENT;
+_ctrlTreeRecent ctrlAddEventHandler ["TreeSelChanged", {
+    params ["_ctrlTreeRecent", "_selectedPath"];
+
+    // Store data of selected item to allow for deleting the of crew of objects placed through the recent tree
+    // tvCurSel is unavailable once the selected item has been placed, the empty path check ensures that the
+    // data is not cleared since this event occurs before the object placed event
```

Comment:
Maybe just a multi-line comment?

Reply:
This is a pretty standard way of doing a comment over multiple lines. Usually easier depending on editor as well.

Reply:
I prefer doing it like this over using `/* */`.